nautobot/extras/templates/extras/note.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load buttons %}
+{% load plugins %}
+{% load perms %}
+{% load helpers %}
+
+{% block breadcrumbs %}
+    <li><a href="{% url 'extras:note_list' %}">Notes</a></li>
+    {% if object.assigned_object.get_absolute_url %}
+        <li><a href="{{ object.assigned_object.get_absolute_url }}changelog/">{{ object.assigned_object }}</a></li>
+    {% endif %}
+    <li>{{ object }}</li>
```

Comment:
Is this correct? Feels weird to have the changelog URL in the breadcrumbs for a Note.

Reply:
Copy/Paste issue

Reply:
Fixed!